app/backend/src/couchers/models.py

```diff
@@ -252,6 +258,21 @@ def __repr__(self):
         return f"LoginToken(token={self.token}, user={self.user}, created={self.created}, expiry={self.expiry})"
 
 
+class PasswordResetToken(Base):
+    __tablename__ = "password_reset_tokens"
+    token = Column(String, primary_key=True)
+
+    user_id = Column(Integer, ForeignKey("users.id"), nullable=False)
+
+    created = Column(DateTime(timezone=True), nullable=False, server_default=func.now())
+    expiry = Column(DateTime(timezone=True), nullable=False)
```

Comment:
Describe that the timezone should always be UTC (and I think `func.now()` violates that condition)